.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: 🐜 Bug Report
+description: Tell us about something not working properly
+labels: [ "kind/bug", "priority/needs-triage" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Bug Report
+        
+        Thanks for helping make the Kubeflow project better!
+
+  - type: checkboxes
+    attributes:
+      label: Checks
+      options:
+        - label: I have searched the [existing issues](https://github.com/kubeflow/dashboard/issues).
+          required: true
+        - label: My issue is related to one of the components in the [`kubeflow/dashboard`](https://github.com/kubeflow/dashboard) repository.
+          required: true
+
+  - type: input
+    id: kubeflow-version
+    attributes:
+      label: Kubeflow Version
+      placeholder: X.X.X
+    validations:
+      required: true
+
+  - type: input
+    id: kubeflow-distribution
+    attributes:
+      label: Kubeflow Distribution
+      description: How are you deploying Kubeflow Platform?
+      placeholder: ...
+    validations:
+      required: true
```

Comment:
```suggestion
  - type: input
    id: kubeflow-installation-method
    attributes:
      label: Installation Method
      description: How are you deploying Kubeflow Platform? For explanation refer to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/)
      placeholder: ...
    validations:
      required: true
```

Reply:
made similar (but shorter text) modifications in https://github.com/kubeflow/dashboard/pull/75/commits/a48f1ca9de3c2146c2f8158d98b567b63592726b